torch/_inductor/overrides.py

```diff
@@ -559,18 +560,75 @@ def fuse_fx(gm: torch.fx.GraphModule, example_inputs):
     return gm
 
 
+# check the pattern: (nn.module, F.function) matched.
+def matches_module_function_pattern(pattern, node, modules):
+    if len(node.args) == 0:
+        return False
+    if not isinstance(node.args[0], torch.fx.Node) or not isinstance(
+        node, torch.fx.Node
+    ):
+        return False
+    # the first node is call_module
+    if node.args[0].op != "call_module":
```

Comment:
I think we also need to check that `len(node.args[0].users) == 1`, unless you support the multiple output case.  This might be true for some of the other pattern matchers too.

Reply:
@jansel , there has a users check at line 651: 
```
            if matches_module_function_pattern(pattern, node, modules):
                if len(node.args[0].users) > 1:  # Output of conv is used by other nodes
```